### PR TITLE
istioctl: improve  admin log flag description

### PR DESCRIPTION
**Pull request by my-git9**
**Please provide a description of this PR:**

istioctl: improve  admin log flag description

### CODE REVIEW

**Review by ymesika — APPROVED** [review]
LGTM

File: istioctl/pkg/admin/istiodconfig.go [hunk 1]
```diff
@@ -483,10 +483,10 @@ func istiodLogCmd(ctx cli.Context) *cobra.Command {
 	logCmd.PersistentFlags().BoolVar(&istiodReset, "reset", istiodReset, "Reset levels to default value. (info)")
 	logCmd.PersistentFlags().IntVar(&controlzPort, "ctrlz_port", 9876, "ControlZ port")
 	logCmd.PersistentFlags().StringVar(&outputLogLevel, "level", outputLogLevel,
-		"Comma-separated list of output logging level for scopes in format <scope>:<level>[,<scope>:<level>,...]"+
+		"Comma-separated list of output logging level for scopes in format <scope>:<level>[,<scope>:<level>,...] "+
```
Comment (ymesika) [inline istioctl/pkg/admin/istiodconfig.go:486]:
```suggestion
		"Comma-separated list of output logging level for scopes in the format of <scope>:<level>[,<scope>:<level>,...] "+
```

File: istioctl/pkg/admin/istiodconfig.go [hunk 2]
```diff
@@ -483,10 +483,10 @@ func istiodLogCmd(ctx cli.Context) *cobra.Command {
 	logCmd.PersistentFlags().BoolVar(&istiodReset, "reset", istiodReset, "Reset levels to default value. (info)")
 	logCmd.PersistentFlags().IntVar(&controlzPort, "ctrlz_port", 9876, "ControlZ port")
 	logCmd.PersistentFlags().StringVar(&outputLogLevel, "level", outputLogLevel,
-		"Comma-separated list of output logging level for scopes in format <scope>:<level>[,<scope>:<level>,...]"+
+		"Comma-separated list of output logging level for scopes in the format of <scope>:<level>[,<scope>:<level>,...] "+
```
Comment (hanxiaop) [inline istioctl/pkg/admin/istiodconfig.go:486]:
```suggestion
		"Comma-separated list of output logging level for scopes in the format of <scope>:<level>[,<scope>:<level>,...]. "+
```

File: istioctl/pkg/admin/istiodconfig.go [hunk 3]
```diff
@@ -483,10 +483,10 @@ func istiodLogCmd(ctx cli.Context) *cobra.Command {
...
 			"Possible values for <level>: none, error, warn, info, debug")
 	logCmd.PersistentFlags().StringVar(&stackTraceLevel, "stack-trace-level", stackTraceLevel,
-		"Comma-separated list of stack trace level  for scopes in format <scope>:<stack-trace-level>[,<scope>:<stack-trace-level>,...] "+
+		"Comma-separated list of stack trace level for scopes in the format of <scope>:<stack-trace-level>[,<scope>:<stack-trace-level>,...] "+
```
Comment (hanxiaop) [inline istioctl/pkg/admin/istiodconfig.go:489]: same as above

```suggestion
		"Comma-separated list of stack trace level for scopes in the format of <scope>:<stack-trace-level>[,<scope>:<stack-trace-level>,...]. "+
```

**Review by hanxiaop — APPROVED** [review]
LG

**Pull request merged**